src/preferences/dialog/dlgprefwaveformdlg.ui

```diff
@@ -330,6 +343,16 @@
        </property>
       </widget>
      </item>
+     <item row="12" column="1">
+      <widget class="QCheckBox" name="beatGridLinesCheckBox">
+       <property name="toolTip">
+        <string>Display beat grid lines on waveform</string>
```

Comment:
I will make these changes, and look into why the preference is being ignored when mixxx is restarted.